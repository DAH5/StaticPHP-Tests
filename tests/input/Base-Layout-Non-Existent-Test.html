---
layout: tests/input/_includes/non-existent-layout.html
---
<p>Testing that a non-existent base layout path does not result in anything other than this line.</p>