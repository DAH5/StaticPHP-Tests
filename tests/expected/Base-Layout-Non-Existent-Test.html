<p>Testing that a non-existent base layout path does not result in anything other than this line.</p>